src/Neo/Wallets/Wallet.cs

```diff
@@ -578,99 +578,12 @@ private Transaction MakeTransaction(DataCache snapshot, ReadOnlyMemory<byte> scr
                     tx.SystemFee = engine.GasConsumed;
                 }
 
-                tx.NetworkFee = CalculateNetworkFee(snapshot, tx, maxGas);
+                tx.NetworkFee = tx.CalculateNetworkFee(snapshot, ProtocolSettings, (a) => GetAccount(a)?.Contract?.Script, maxGas);
```

Comment:
isn´t this function a little complex `ProtocolSettings, (a) => GetAccount(a)?.Contract?.Script`?
Maybe we should just pass the Script

Reply:
There can be a lot of scripts